gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/setPayees.ts

```diff
@@ -100,20 +102,44 @@ export default class SetPayees extends SolanaCommand {
 
     logger.log('Payees information:', input)
     await prompt('Continue setting payees?')
+    const data = program.coder.instruction.encode('set_payees', {
+      payees,
+    })
 
-    const tx = await program.rpc.setPayees(payees, {
-      accounts: {
-        state: state,
-        authority: owner.publicKey,
+    const accounts: AccountMeta[] = [
+      {
+        pubkey: state,
+        isSigner: false,
+        isWritable: true,
       },
-      signers: [owner],
-    })
+      {
+        pubkey: signer,
+        isSigner: true,
+        isWritable: false,
+      },
+    ]
 
+    const rawTx: RawTransaction = {
+      data,
+      accounts,
+      programId: ocr2.programId,
+    }
+
+    return [rawTx]
+  }
+
+  execute = async () => {
+    const rawTx = await this.makeRawTransaction(this.wallet.payer.publicKey)
+    const tx = makeTx(rawTx)
+    logger.debug(tx)
+    logger.loading('Sending tx...')
+    const txhash = await this.sendTx(tx, [this.wallet.payer], this.idl)
```

Comment:
For the moment let's load the contract and program here too. Open an issue please to circle back here. You can use `program.idl` then